### PR TITLE
Update schedule

### PR DESCRIPTION
**Pull request by ryyppy**

### CODE REVIEW

**Comment by okonet** [comment]
I don’t support such type now. Please export panel discussion as a talk

**Comment by okonet** [comment]
Argh as an event like lunch

**Comment by okonet** [comment]
What about the placeholder? Did someone drop out?

**Comment by ryyppy** [comment]
@okonet I adapted the type to "PANEL", as discussed with @bebraw

File: common/exporter.re [hunk 1]
```diff
@@ -100,6 +102,9 @@ let convertTask = (task: Data.Timetable.task): array(session) =>
   | OpenEnd(title) => [|
       session(~type_="OPEN_END", ~title, ~description="", ()),
     |]
+  | PanelDiscussion => [|
+      session(~type_="PANEL", ~title="Panel Discussion", ~description="", ()),
+    |]
   | Misc(title) => [|session(~type_="MISC", ~title, ~description="", ())|]
   | _ => [|session(~type_="UNKNOWN", ~title="", ~description="", ())|]
```
Comment (bebraw) [inline common/exporter.re]: Is this one correct? Should the panel replace this one?

Comment (bebraw) [reply common/exporter.re]: Ah, never mind. This is just the exporting portion. It's ok.

**Pull request merged**